v3/src/components/case-table/use-columns.tsx

```diff
@@ -25,6 +27,28 @@
   return formatter
 }
 
+export function renderValue(str = "", num = NaN, attr?: IAttribute) {
+  const type = attr?.type
+
+  // colors
+  if (type === "color" && parseColor(str).space) {
```

Comment:
Looks like if the type is not "color" the swatch will never show. But we want "strict" colors to display as swatches always, right?

Reply:
That wasn't my understanding from our previous conversation, but we can discuss at the planning meeting today.